Makefile

```diff
@@ -21,6 +21,10 @@ build-dist: build
 	node scripts/build-dist.js
 	node scripts/generate-babel-types-docs.js
 
+build-babel-standalone:
+	cd packages/babel-standalone; \
```

Comment:
I think this will mess up the cwd, since you don't `cd ../..` at the end. It should likely use `pushd` and `popd` instead. Having said that, Gulp has a `--cwd` option that might be able to be used instead: https://github.com/gulpjs/gulp/blob/master/docs/recipes/specifying-a-cwd.md

eg. 
```
./node_modules/.bin/gulp build --cwd=packages/babel-standalone
```